subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -201,7 +201,7 @@ static struct {
 
 static struct {
 	void *free;
-	uint8_t pool[sizeof(memq_link_t) * EVENT_DONE_MAX];
+	uint8_t pool[sizeof(memq_link_t) * (EVENT_DONE_MAX + 1)];
```

Comment:
```suggestion
	uint8_t pool[sizeof(memq_link_t) * (EVENT_DONE_MAX + EVENT_DONE_LINK_CNT)];
```

define `EVENT_DONE_LINK_CNT` next to `EVENT_DONE_MAX` in lll.h file. Apply `CONFIG_BT_CTLR_LOW_LAT_UL` conditional.